tests/monotouch-test/monotouch-test.csproj

```diff
@@ -790,7 +791,6 @@
   </ItemGroup>
   <ItemGroup>
     <EmbeddedResource Include="Security\openssl_crt.der">
-        <LogicalName>monotouchtest.Security.openssl_crt.der</LogicalName>
```

Comment:
Unintended change?

Reply:
Looks like VS does that. Will revert.

Reply:
Woow, glad we have diffs, it is really scary that VS4Mac is deleting stuff on your back

Reply:
It did the same thing in my metal PR